Cgl for tables

michelmno opened the pull request:
Cgl for tables, changes splitted in 4 commits to ease reading

* Put back cgl button for table view 
  This was previously commented out by
  cid 9f7c755a8 "new minimal generation v0 parameter for tables
  probably an un-noticed side effect.
    
* buttons.txt cgl table view in new browser tab
  as already done for cgl tree view in menubar.txt

* Do not display note and source if cgl=on
  for ascendant an descendant tables view

* Do not access detailed img from table if cgl=on
  Now same behavior for descendants table view as ascendant table view.

michelmno committed:
Put back cgl button for table view

This was previously commented out by
cid 9f7c755a8 "new minimal generation v0 parameter for tables
probably an un-noticed side effect.

Signed-off-by: Michel Normand <michel_mno@laposte.net>

michelmno committed:
Do not display note and source if cgl=on

for ascendant an descendant tables view

Signed-off-by: Michel Normand <michel_mno@laposte.net>

michelmno committed:
buttons.txt cgl table view in new browser tab

as already done for cgl tree view in menubar.txt

Signed-off-by: Michel Normand <michel_mno@laposte.net>

michelmno committed:
Do not access detailed img from table if cgl=on

Now same behavior for descendants table view as ascendant table view.

Signed-off-by: Michel Normand <michel_mno@laposte.net>

a2line commented:
Hi Michel.

About first commit: Why do you want the button back in this toolbar? It’s now redundant with the generic one that does exactly the same in menubar (you removed that comment yourself in this commit). Also we don’t put it on every toolbar and there's, moreover, already the plugin ‘cgl’ that does the same too. That was a bit redundant, so I commented it to lighten the already toolbar that needed space for accessibility. Of course, we can put it back if it’s used a lot.

Third commit: Why do you remove sources and notes markers if ’ns=1’, it’s not logical here. One can go to print them to know the state of the art, I understand the popup could be removed but that interactive popup doesn’t show on printed resources, do you have a use case where this is problematic? The cgl mode was made originally to print data removing unneeded blue links, here this does not pose any problem. So I think cgl could remove the popups, but not the markers that are in corners, because if you don’t want them you set ’ns=0’ that is no evar.ns in URL.

michelmno committed:
Do not display update/modify message if cgl=on

And
do not allow navigation individual change if cgl=on
both for for ascendant/descendant tree views.

Signed-off-by: Michel Normand <michel_mno@laposte.net>

hgouraud commented:
I dont see the benefit of this PR against the generig CGL button in menubar
Notes and sources markers should be kept, with their colors

michelmno commented:
closed as not accepted
